test/plugins/plugins-document.skip-test.js

```diff
@@ -19,34 +21,56 @@ beforeEach(async (t) => {
 });
 
 test('Basic document template is read and loaded', async (t) => {
-  const { app } = t.context;
+  const app = /** @type {import('fastify').FastifyInstance} */ (t.context.app);
   await writeFile(
     join(tmp, 'document.js'),
     'export default (incoming, body) => `hello world`;',
   );
   const podlet = new Podlet({ name: 'test', version: '1.0.0', pathname: '/' });
   await app.register(plugin, { cwd: tmp, development: true, podlet });
-  // @ts-ignore
-  t.equal(
-    podlet._view(),
-    'hello world',
+  await app.register(fastifyPodlet, podlet);
+  app.get('/', async (request, reply) => {
+    // @ts-ignore
+    return reply.podiumSend('');
+  });
+  const response = await app.inject({
+    method: 'GET',
+    url: '/',
+    headers: {
+      accept: 'text/html',
+    },
+  });
+  t.match(
+    response.body,
+    /hello world/,
```

Comment:
`response.body` is an empty string here, which is unexpected. Logging confirms we do set `podium.view` to a function in the document plugin on L30 😕